src/deployment/deploy.py

```diff
@@ -1147,6 +1147,10 @@ def expand_agent(f: str) -> List[str]:
                             ],
                             env=dict(os.environ, CLI_DEBUG="1"),
                         )
+
+                        if function_name == "queue_proxy_update":
```

Comment:
why the change of name ?

Reply:
I changed it awhile back because I thought it was a more accurate function name. The function _only_ handles the heartbeat updates.

Reply:
i propose we keep it as is and change it after the migration is done.

Reply:
this will minimize the custom code we need to support the rename

Reply:
Done.